codegen/src/test/resources/software/amazon/awssdk/codegen/poet/client/c2j/json/customization.config

```diff
@@ -8,5 +8,10 @@
     "verifiedSimpleMethods" : ["paginatedOperationWithResultKey"],
     "blacklistedSimpleMethods" : [
         "eventStreamOperation"
-    ]
+    ],
+  "utilitiesMethod": {
+    "name": "utilities",
+    "returnType": "software.amazon.awssdk.services.json.JsonUtilities",
+    "createMethodParams": ["param1", "param2", "param3"]
```

Comment:
Should the params just be hard-coded to be the sdk configuration?

Reply:
I want to keep it configurable as we can't predict that future utilities just take sdk configuration.

Reply:
Since this is internal, can we just YAGNI it and add it if we need in the future?

Reply:
why to remove a supported feature? As its internal, changes to it shouldn't affect customers.

Reply:
I think another option would be to provide a params object (`ClientUtilityParams`?) instead so we don't need to expand the params.